prog/prio.go

```diff
@@ -196,16 +196,20 @@ func (target *Target) BuildChoiceTable(corpus []*Prog, enabled map[*Syscall]bool
 		}
 	}
 	noGenerateCalls := make(map[int]bool)
+	enabledCalls := make(map[*Syscall]bool)
 	for call := range enabled {
-		if call.Attrs.Disabled {
-			delete(enabled, call)
-		} else if call.Attrs.NoGenerate {
+		if call.Attrs.NoGenerate {
 			noGenerateCalls[call.ID] = true
-			delete(enabled, call)
+		} else if !call.Attrs.Disabled {
```

Comment:
Let's merge all changes like this separately before the main change. It will simplify review/updates of the larger changes.